browser/src/map/Map.js

```diff
@@ -479,7 +543,12 @@ L.Map = L.Evented.extend({
 				timeout = 60000;
 			}
 
-			this.lastModIndicator.innerHTML = dateValue;
```

Comment:
we also could do `this.fire('lastmodificationupdate', some data)` - then we can handle it in many places by:
`map.on('lastmodificationupdate', this.someHandlerFunction, this);`  so `someHandlerFunction` will do the job - without tight coupling the code